src/harness/testcases/testdata/device_h.json

```diff
@@ -4,8 +4,6 @@
   "cbsdSerialNumber": "test_serial_number_h",
   "callSign": "callsign_h",
   "cbsdCategory": "B",
-  "measCapability": [
-      ],
   "airInterface": {
```

Comment:
why are we removing measCapability from device_h? it's not used by this PR but won't it affect other PRs that use device_h?

Reply:
Sorry, was a mistake. Please take a look.